### PR TITLE
P#69855 Fix duplicate check warning

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

File: plugin/Controller/AdminViewController.php [hunk 1]
```diff
@@ -311,17 +311,15 @@ public function enqueue_css()
 
 	public function enqueueExtraJs($hook)
 	{
-		if (__String::getNew($hook)->contains('onoffice')) {
```
Comment (LongTrong-exe) [inline plugin/Controller/AdminViewController.php:314]: Removing the if condition here will result in `$pObject->doExtraEnqueues();` will be called on every page.

**Review by LongTrong-exe — COMMENTED** [review]
According to the code, requirment 1 has not been fixed.

File: plugin/Controller/AdminViewController.php (hunk 1 above)
Comment (hungnc89) [reply plugin/Controller/AdminViewController.php:314]: That's right, cause the warning is appeared on every pages.

**Review by LongTrong-exe — APPROVED** [review]

**Review by jmaas-onoffice — APPROVED** [review]

**Comment by hungnc89** [comment]
- Tester tested
- Video:

https://user-images.githubusercontent.com/72601287/168030730-2dce8fb1-c54b-4285-84b1-f75ed94a8d49.mp4

**Pull request merged**